mobile/library/objective-c/proxy/EnvoyPACProxyResolver.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
```

Comment:
do we need any of these files (the objective-c implementation)?  i don't think so... i think the c++ implementation plus calling the register_apple_proxy_resolver ( that triggers everything) from the objective-c EnvoyEngineImpl when enableProxying(true) is sufficient..